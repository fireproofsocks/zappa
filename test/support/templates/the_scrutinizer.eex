<div>
  My name is <%= HtmlEntities.encode(name) %>. I am a <%= HtmlEntities.encode(occupation) %>.
</div>