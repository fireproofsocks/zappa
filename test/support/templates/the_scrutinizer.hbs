<div>
  Desperate {{people}} in high offices all over the world have been known to
  enact the most disgusting pieces of {{output}} in order to win {{goal}}
  (or, in places where they don't get to vote, to control unwanted forms of
  {{uncontrollable}}).
</div>