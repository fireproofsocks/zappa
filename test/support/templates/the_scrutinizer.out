<div>
  Desperate nerds in high offices all over the world have been known to
  enact the most disgusting pieces of legislation in order to win votes
  (or, in places where they don't get to vote, to control unwanted forms of
  mass behavior).
</div>